### PR TITLE
feat: Setup Spring boot application

### LINKED ISSUE #1: Initial Set-up

### LINKED ISSUE #4: Java: Setup spring-boot app

### PR DESCRIPTION
**Pull request by dricazenck**
Setup the spring boot application. 

close #1

### CODE REVIEW

File: .idea/google-java-format.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="GoogleJavaFormatSettings">
+    <option name="enabled" value="false" />
```
Comment (dricazenck) [inline .idea/google-java-format.xml:4]: @goelsonali what do you think of us using google format, we can also adjust the code format if we want to.

**Review by kikilondon — APPROVED** [review]

File: build.gradle.kts
Comment (nverbic) [inline build.gradle.kts:1]: Just one observation (might be wrong, becuase I am just starting with hello world setup here :-)), do we need also this dependency below: Springdoc OpenAPI UI dependency?

**Review by goelsonali — APPROVED** [review]

File: .idea/google-java-format.xml (hunk 1 above)
Comment (goelsonali) [reply .idea/google-java-format.xml:4]: i think ideas are good enough to format the code, I think we don't need these format settings.
But would like to explore SONAR option how to configure SONAR and run default checksum rules.

File: build.gradle.kts
Comment (dricazenck) [reply build.gradle.kts:1]: I am not sure what do you mean?

File: .idea/google-java-format.xml (hunk 1 above)
Comment (dricazenck) [reply .idea/google-java-format.xml:4]: yep, we can create issues for that and take care soon

File: build.gradle.kts
Comment (nverbic) [reply build.gradle.kts:1]: When I was researching how to setup the new Sping Boot project locally (on Windows) I have come across this dependency. But now everything is working well without this line so I would mark this comment as not relevant.

**Pull request merged**